lnst/RecipeCommon/Perf/Evaluators/BaselineEvaluator.py

```diff
@@ -53,8 +58,23 @@ def evaluate_group_results(
             )
             comparison_result = ResultType.max_severity(comparison_result, comparison)
             result_text.extend(text)
+            comparisons.append(
+                {
+                    "current_result": result,
+                    "baseline_result": baseline,
+                    "comparison_result": comparison,
```

Comment:
wouldn't be something like `run_result` more descriptive? `comparison_result` is also a variable used few lines above that represents cumulative result

Reply:
yeah i was debating that, but i'm not sure what else to call a result of a single comparison, maybe it would make sense to rename the other vars instead?
* comparison_result for a single comparison
* cumulative_result for the max_severity result (what is now called comparison_result)

Reply:
I think we should be careful not to confuse this variable with possible reference to Catalog run as well, even though it has nothing to do with Catalog just yet.

Reply:
i did the renaming here so i think this should be ok.

@Axonis  what do you mean by Catalog confusion?

Reply:
`run_result` could refer to a single run / iteration or a single test run or even catalog run object, that's why I wanted to avoid it.

Reply:
well, that's a different context though. from the point of view of the `BaselineEvaluator` you only have a current result and the baseline result, both of which are some kind of a `*PerfMeasurementResult` type.

either way, is the current state ok and can we resolve this thread or do you propose a different naming?

Reply:
Yes, I am aware of that.

I am okay with current name.